<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplicationSubscriberWizardPage</class>
 <widget class="QWizardPage" name="ReplicationSubscriberWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="subscribersL">
       <property name="text">
        <string>Subscribers:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeSubscriberPB">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="subscribersLW"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="newSubscriberL">
       <property name="text">
        <string>New subscriber:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newSubscriberLE"/>
     </item>
     <item>
      <widget class="QPushButton" name="addSubscriberPB">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addSubscriberPB</sender>
   <signal>clicked()</signal>
   <receiver>ReplicationSubscriberWizardPage</receiver>
   <slot>addSubscriberPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSubscriberPB</sender>
   <signal>clicked()</signal>
   <receiver>ReplicationSubscriberWizardPage</receiver>
   <slot>removeSubscriberPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeSubscriberPB_clicked()</slot>
  <slot>addSubscriberPB_clicked()</slot>
 </slots>
</ui>
